ephios/plugins/questionnaires/forms.py

```diff
@@ -32,9 +32,9 @@ def __init__(self, data=None, **kwargs):
 
     def save(self, commit=True):
         self.instance.choices = [
-            choice["name"]
-            for choice in self.choices.cleaned_data
-            if "name" in choice and not choice["DELETE"]
+            choice_form_data["name"]
+            for choice_form_data in self.choices.cleaned_data
+            if choice_form_data and not choice_form_data["DELETE"]
```

Comment:
Now I don't get this check. Why would `choice_form_data` ever be falsy? Doesn't it always contain at least the `DELETE` key?